integration_tests/src/main/python/row-based_udf_test.py

```diff
@@ -1,43 +0,0 @@
-# Copyright (c) 2021, NVIDIA CORPORATION.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import pytest
-
-from asserts import assert_gpu_and_cpu_are_equal_sql
-from data_gen import *
-from spark_session import with_spark_session
-from rapids_udf_test import skip_if_no_hive, load_hive_udf_or_skip_test
-
-def test_hive_empty_simple_udf():
-    with_spark_session(skip_if_no_hive)
-    data_gens = [["i", int_gen], ["s", string_gen]]
-    def evalfn(spark):
-        load_hive_udf_or_skip_test(spark, "emptysimple", "com.nvidia.spark.rapids.udf.hive.EmptyHiveSimpleUDF")
```

Comment:
This file should not be deleted.  It is not testing RAPIDS accelerated UDFs but CPU UDFs accelerated by a feature built into the RAPIDS Accelerator plugin.  The degenerate Hive UDFs used for these tests should be moved to the integration_tests jar, as they are not real-world examples we would want users to emulate.